specs/3/README.md

```diff
@@ -0,0 +1,620 @@
+---
+
+slug: CS-03
+title: CS-03/EXCUBIAE-V0.3.0
```

Comment:
Nitpick: But I'd rather we call this `CS-03/EXCUBIAE`. If we want to have versions in this it should either be major versions, e.g. Excubiae v1 or v2 with their own spec, or it should be an evolving spec. E.g. currently it is 0.3, but in future it is 0.4 and the draft spec updates with it. To track changes between spec versions this can be done with a simple protocol changelog. E.g. see https://rfc.vac.dev/waku/standards/legacy/6/waku1#changelog

Reply:
I'll commit this. Feel free to add v0.3 and changelog logic in follow up PR.